server/handler/assignments.go

```diff
@@ -45,16 +49,22 @@ type assignmentRequest struct {
 }
 
 // SaveAssignment returns a function that saves the user answers as passed in the body request
-func SaveAssignment() RequestProcessFunc {
+func SaveAssignment(repo *repository.Assignments) RequestProcessFunc {
 	return func(r *http.Request) (*serializer.Response, error) {
-		requestedAssignmentID := chi.URLParam(r, "assignmentId")
-		assignmentID, err := strconv.Atoi(requestedAssignmentID)
+		requestedPairID := chi.URLParam(r, "pairId")
+		pairID, err := strconv.Atoi(requestedPairID)
```

Comment:
I see we do it often. Just an idea for the future:

```
func URLParamInt(r, key) (int, error) {
  str := chi.URLParam(r, "pairId")
  return strconv.Atoi(str)
}
```

Reply:
done in 01f1cbc